### PR TITLE
W-16249004-executiontimetableinformation-se

### PR DESCRIPTION
**Pull request by sebermuley**
Added information that only calendar scheduled run configurations are shown in the timetable and addressed some Vale suggestions.

### CODE REVIEW

**Comment by sebermuley** [comment]
Dear SME, please review line 18 and approve this PR. Thank you.

**Comment by sebermuley** [comment]
UI Text in https://www.figma.com/board/bJgwx6D39BLuK1niN5uE5H/%5BUI-Text%5D-RPA---Execution-Timetable?node-id=0-1&t=HaRHI7QTBwWtWmCt-0

**Review by jhammann72 — DISMISSED** [review]

File: rpa-manager/modules/ROOT/pages/botmanagement-troubleshoot.adoc [hunk 1]
```diff
@@ -10,20 +10,19 @@ If a process execution fails when a bot runs correctly, turn on debug logging to
...
 == View the Planned Process Executions
 
-If you are not sure why a bot is not processing, review the execution plans:
+If you're not sure why a bot isn't processing, review the execution plans:
 
 . In the module *Bot Management*, open the view *Bot Execution Plans*.
 . In the table, click *Execution timetable* (image:executiontimetable-icon.png[calendar symbol,1.5%,1.5%]) in the row of the bot you want to check.
 
-The execution timetable shows you the effective runtime of all configurations assigned to the bot.
+The execution timetable shows you the effective runtime of all calendar scheduled run configurations assigned to the bot. The execution timetable does not show run configurations using other types of scheduling, like on-demand, interval or cron-expression.
```
Comment (Cristian-Venticinque) [inline rpa-manager/modules/ROOT/pages/botmanagement-troubleshoot.adoc]:
```suggestion
The execution timetable shows the effective runtime of all calendar scheduled run configurations assigned to the bot. The execution timetable does not show run configurations using other types of scheduling, such as on-demand, interval, or cron-expression.
```

File: rpa-manager/modules/ROOT/pages/botmanagement-troubleshoot.adoc [hunk 2]
```diff
@@ -10,20 +10,19 @@ If a process execution fails when a bot runs correctly, turn on debug logging to
...
 
 . Click *Configure Execution Timetable* (image:configure-icon.png[wrench symbol,1.5%,1.5%]) to change the time interval of the table and to select the shown configurations.
 . Click *OK*.
 . In the *Execution Timetable*, click a colored bar in the row of the configuration you want to check to view the composition of the effective runtime.
+The bot runs a process when the bot's service time overlaps with the schedule of the configuration. If the process uses applications, the bot doesn't run it during the downtimes of any of those applications.
```
Comment (Cristian-Venticinque) [inline rpa-manager/modules/ROOT/pages/botmanagement-troubleshoot.adoc]:
```suggestion
The bot runs a process when the bot's service time overlaps with the configuration schedule. If the process uses applications, the bot doesn't run the process during the downtimes of any of those applications.
```

**Review by Cristian-Venticinque — DISMISSED** [review]

**Review by Cristian-Venticinque — APPROVED** [review]

**Pull request merged**